src/main/resources/assets/tconstruct/book/materials_and_you/en_us/modifiers/emerald.json

```diff
@@ -0,0 +1,13 @@
+{
+  "modifier_id": "tconstruct:emerald",
+  "text": [
+    {
+      "text": "A villager's best friend, don't let the villagers see you applying this! These aren't your lucky charms."
+    }
+  ],
+  "effects": [
+    "Increases tool durability depending on base durability",
```

Comment:
Also gives bonus damage against illagers. Yeah, was not sure where else to put that bonus